spec/spec_helper_acceptance.rb

```diff
@@ -22,14 +22,32 @@
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
+      if fact_on(host, 'os.family') == 'RedHat'
```

Comment:
Please submit this to the template.
https://github.com/theforeman/foreman-installer-modulesync/blob/1b9f93ba4c701387945e279968c491bb26470009/moduleroot/spec/spec_helper_acceptance.rb.erb#L30

Reply:
Will do shortly, thanks